src/components/charts/colorMixingTool/index.jsx

```diff
@@ -1,83 +1,83 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import Button from '../../site/button/index.js';
-import { redBtn } from '../../site/button/styles.module.scss';
 import {
-  groupLayout,
   colorImage,
   button,
-  buttonClicked,
+  active,
+  col50,
+  imagesContainer,
+  buttonContainer,
 } from './color-tool.module.scss';
-import greenPic from '../../../../static/images/green-filter.png';
-import redPic from '../../../../static/images/red-filter.png';
-import bluePic from '../../../../static/images/blue-filter.png';
 
 class ColorTool extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       showImage: false,
       imgSrc: null,
-      greenIsClick: false,
-      redIsClick: false,
-      blueIsClick: false,
+      greenIsActive: false,
+      redIsActive: false,
+      blueIsActive: false,
     };
   }
 
-  handleImage(color, clickValue) {
-    const src =
-      color === 'red' ? redPic : color === 'blue' ? bluePic : greenPic;
+  handleImage(color, src) {
+    const stateName = color + 'IsActive';
     this.setState(prevState => ({
       ...prevState,
       showImage: true,
```

Comment:
for your `showImage: true` you can try setting it to `showImage: !prevState[stateName],`